vault/resource_database_secrets_mount.go

```diff
@@ -197,6 +197,11 @@ func getCommonDatabaseSchema() schemaMap {
 				Type: schema.TypeString,
 			},
 		},
+		"password_policy": {
```

Comment:
We should set Computed: True since this returns a default if not set. That will allow TFVP to fallback to the value returned by Vault.

https://developer.hashicorp.com/vault/api-docs/secret/databases#password_policy

Reply:
Thanks for having a preliminary look @fairclothjm. Much appreciated!

I have a limited familiarity with Vault's source code, but it seems that the default password policy is never externally exposed. From a client's perspective, the default password policy is an empty string. Does that align with your reasoning for setting `Computed: true`?

***

Enabling the computed behavior resolves the drift initially observed, but the attribute will quietly remain at the previously applied value. I believe such a scenario (set and applied -> removed from configuration) would be rare, but in my opinion, it makes for an obscure UX.